tools/tests/rangeslider/test.js

```diff
@@ -121,6 +137,109 @@ describe('Rangeslider component', () => {
         assert.equal(left, value);
     });
 
+    it('Should change value dynamically', async () => {
+        await loadRangeslider({});
+        const rangeSlider = document.querySelector('gameface-rangeslider');
+        const rangesliderHandle = document.querySelector('.guic-horizontal-rangeslider-handle');
+        rangeSlider.value = 70;
+        let left = parseFloat(rangesliderHandle.style.left);
+        assert.equal(left, 70);
+
+        rangeSlider.setAttribute('value', 20);
+        left = parseFloat(rangesliderHandle.style.left);
+        assert.equal(left, 20);
+
+        rangeSlider.value = 'invalid';
+        left = parseFloat(rangesliderHandle.style.left);
+        assert.equal(left, 0);
+    });
+
+    it('Should change value dynamically that overflows min and max', async () => {
+        await loadRangeslider({ min: 10, max: 70 });
+        const rangeSlider = document.querySelector('gameface-rangeslider');
+        const rangesliderHandle = document.querySelector('.guic-horizontal-rangeslider-handle');
+        rangeSlider.value = 200;
+        let left = parseFloat(rangesliderHandle.style.left);
+        assert.equal(left, 100);
+
+        rangeSlider.setAttribute('value', -10);
+        left = parseFloat(rangesliderHandle.style.left);
+        assert.equal(left, 0);

```

Comment:
Same as before and any other

Reply:
Split the test into both for min and max as you suggested.